Virtualization Survey 20251111

Cyanoxygen opened the pull request:
Topic Description
-----------------

- open-vm-tools: update to 13.0.5
- virt-manager: update to 5.1.0
    \* Refresh patches.
- libvirt-python: update to 11.9.0
    - Use PyPI source and correct Anitya \(the old ID references a file server
    that does not seem to sync with new releases reliably\).
    - Use pep517 ABTYPE.
- libvirt: update to 11.9.0
    \* Strip out HTML documentation.
    \* Refresh patches.
- virglrenderer: update to 1.2.0
    \* Enable Venus \(Vulkan\) renderer.
    \* Disable minigbm_allocation due to reports of "broken SPICE screen" with
    AMD graphics cards \(see link\).
    \* Clean up Meson options and comments.
    \* Add pyyaml as build dependency.
    \* Convert MESON_AFTER to array.
    Link: https://github.com/AOSC-Dev/aosc-os-abbs/pull/13825#discussion_r2573308089
- wireshark: update to 4.6.1
    Co-authored-by: 白铭骢 \(Mingcong Bai\) \(@MingcongBai\) <jeffbai@aosc.io>
- virtiofsd: update to 1.13.2

Package(s) Affected
-------------------

- libvirt: 11.9.0
- open-vm-tools: 13.0.5
- virglrenderer: 1.2.0
- virtiofsd: 1.13.2
- wireshark: 4.6.1

Security Update?
----------------

No

Build Order
-----------

```
#buildit virtiofsd wireshark virglrenderer libvirt libvirt-python virt-manager open-vm-tools
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`
- [ ] LoongArch 64-bit (No SIMD) `loongarch64_nosimd`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Cyanoxygen committed:
libvirt: update to 11.9.0

* Strip out HTML documentation.
* Refresh patches.

Cyanoxygen committed:
open-vm-tools: update to 13.0.5

Cyanoxygen committed:
virt-manager: update to 5.1.0

* Refresh patches.

Cyanoxygen committed:
virglrenderer: update to 1.2.0

* Enable Venus (Vulkan) renderer.
* Disable minigbm_allocation due to reports of "broken SPICE screen" with
  AMD graphics cards (see link).
* Clean up Meson options and comments.
* Add pyyaml as build dependency.
* Convert MESON_AFTER to array.

Link: https://github.com/AOSC-Dev/aosc-os-abbs/pull/13825#discussion_r2573308089

Cyanoxygen committed:
virtiofsd: update to 1.13.2

aosc-buildit-bot committed:
wireshark: update to 4.6.1

Co-authored-by: 白铭骢 (Mingcong Bai) (@MingcongBai) <jeffbai@aosc.io>

misaka4e21 commented on app-virtualization/libvirt/autobuild/defines:
> `-Ddriver_ch=enabled`

Cloud-Hypervisor driver should be disabled to build on architectures other than amd64/arm64.

MingcongBai replied:
I will keep it as is, so long as it builds (we never know).

MingcongBai replied:
Alright fine:

```
meson.build:1579:6: ERROR: Problem encountered: linux on x86_64 or aarch64 is required to build Cloud-Hypervisor driver
```

misaka4e21 commented on app-network/wireshark/spec:
```diff
@@ -1,5 +1,4 @@
-VER=4.4.9
+VER=4.6.0
```
Consider 4.6.1?

MingcongBai replied:
Done.

misaka4e21 commented on app-virtualization/virglrenderer/autobuild/defines:
```diff
@@ -16,14 +16,15 @@ PKGBREAK="qemu<=5.1.0"
-             -Dtracing=none"
+MESON_AFTER=(
+	"-Dplatforms=auto"
+	"-Dminigbm_allocation=true"
```
Broken SPICE screen with `minigbm_allocation` enabled on my computer (AMD 7840HS with Radeon 780M)

This option is now deprecated in the main branch of virglrenderer (though not in the 1.2.0 release)

MingcongBai replied:
Okay, will disable with a FIXME.

misaka4e21 commented on app-virtualization/virglrenderer/autobuild/defines:
```diff
@@ -16,14 +16,15 @@ PKGBREAK="qemu<=5.1.0"
+MESON_AFTER=(
+	"-Dplatforms=auto"
+	"-Dminigbm_allocation=true"
+	"-Dvenus-validate=false"
```
How about enabling venus (`-Dvenus`, probably more testing needed?)

MingcongBai replied:
Done just that, the experimental marker seems to be gone now.

misaka4e21 reviewed (commented):
Thanks for your survey that is far more comprehensive than #10912 and #13821.

I think bumping qemu REL is needed too after rebasing against stable?

MingcongBai committed:
libvirt-python: update to 11.9.0

- Use PyPI source and correct Anitya (the old ID references a file server
  that does not seem to sync with new releases reliably).
- Use pep517 ABTYPE.

MingcongBai commented:
Taking over since @Cyanoxygen has his hands full.

xry111 commented:
<img width="1920" height="1080" alt="屏幕截图_20251201_213305" src="https://github.com/user-attachments/assets/485c1d43-0db3-4403-bc8c-2d85bdecf6b6" />

Smoke tested on LoongArch.
